zerver/webhooks/patreon/view.py

```diff
@@ -0,0 +1,162 @@
+    # first item contains the patron count
+    included = payload["included"][0]
+
+    # get the patron count
+    patron_count = included["attributes"]["patron_count"]
+
+    # create the body
+    body = f"New patron joined! :tada:\nPatrons in total: {patron_count}"
+
+    return body
+
+
+# called when a patron updated their membership
+def members_update(payload: Dict[str, Any]) -> str:
+
+    # array of dicts
+    # first item contains the patron count
+    included = payload["included"][0]
+
+    # get the patron count
+    patron_count = included["attributes"]["patron_count"]
+
+    # create the body
+    body = (
+        f"A patron just updated their membership. :notifications:\nPatrons in total: {patron_count}"
+    )
+
+    return body
+
+
+# called when a patron deleted their membership
+def members_delete(payload: Dict[str, Any]) -> str:
+
+    # array of dicts
+    # first item contains the patron count
+    included = payload["included"][0]
+
+    # get the patron count
+    patron_count = included["attributes"]["patron_count"]
+
+    # create the body
+    body = (
+        f"A patron just deleted their membership. :notifications:\nPatrons in total: {patron_count}"
+    )
+
+    return body
+
+
+# called when a new patron joined through custom pledge
+def members_pledge_create(payload: Dict[str, Any]) -> str:
+
+    # array of dicts
+    # first item contains the patron count
+    included = payload["included"][0]
+
+    # get the patron count
+    patron_count = included["attributes"]["patron_count"]
+
+    # create the body
+    body = f"New patron joined through custom pledge! :tada:\nPatrons in total: {patron_count}"
+
+    return body
+
+
+# called when a patron updated pledge
+def members_pledge_update(payload: Dict[str, Any]) -> str:
+
+    # array of dicts
+    # first item contains the patron count
+    included = payload["included"][0]
+
+    # get the patron count
+    patron_count = included["attributes"]["patron_count"]
+
+    # create the body
+    body = f"A patron just updated their pledge. :notifications:\nPatrons in total: {patron_count}"
+
+    return body
+
+
+# called when a patron deleted pledge
+def members_pledge_delete(payload: Dict[str, Any]) -> str:
+
+    # array of dicts
+    # first item contains the patron count
+    included = payload["included"][0]
+
+    # get the patron count
+    patron_count = included["attributes"]["patron_count"]
+
+    # create the body
+    body = f"A patron just deleted their pledge. :notifications:\nPatrons in total: {patron_count}"
+
+    return body
+
+
+@webhook_view("Patreon")
+@has_request_variables
+def api_patreon_webhook(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    payload: Dict[str, Any] = REQ(argument_type="body"),
+) -> HttpResponse:
+
+    # get the http header
+    event = validate_extract_webhook_http_header(request, "X_PATREON_EVENT", "Patreon")
+    assert event is not None
+
+    # map the event to necessary function
```

Comment:
When you're using a map called `EVENTS_FUNCTION_MAPPER`, this comment becomes pretty redundant.